added electrical section to the read the docs...

ramalamadingdong opened the pull request.

ramalamadingdong committed:
added electrical section to the read the docs...

FromenActual commented:
I was intending for the Control System page to have all the electrical and wiring resources together. There's so much overlap with Control/Expansion Hubs, motors/servos, sensors, wiring, etc. that I think it's better to just keep it all in one place, at least for now while we only have a few resources. If we get a lot more and it makes sense to split it off, we can do that.

FromenActual committed:
Electronics update

Combine Control System and Electrical pages into single Electronics page
Add many resources for electronics

FromenActual commented:
@ramalamadingdong I combined this with the Control System page into a single Electronics page, and added a bunch more resources. Let me know what you think!
